src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -135,8 +135,23 @@ def get_fuzz_task_payload(platform=None):
   if not mappings:
     return None, None
 
+  logs.log(f'Mappings: {mappings}')
+  # The environment variable contains the list of jobs seperated by comma
+  # E.g: "libfuzzer_asan_android_host, afl_asan_android_host, ..."
```

Comment:
@jonathanmetzman is a comma-separated list passed as an env var an acceptable pattern to implement here? It seems potentially error prone, unless we're already doing it in other places throughout ClusterFuzz

Reply:
I think it's fine.